x-pack/heartbeat/monitors/browser/synthexec/synthexec.go

```diff
@@ -108,23 +109,31 @@ func runCmd(
 		return nil, err
 	}
 
+	// Default capabilities
+	capabilities = append(capabilities, "trace", "metrics", "filmstrips", "ssblocks")
```

Comment:
I am not sure if its the right way to do it. 

I was thinking we should make it configurable by introducing Capability flag in the `config.go` in addition to params and then using it here for experimentation. 

moving forward once the feature is built in to the UI, we can add it to `--rich-events` in synthetics. 

WDYT?

Reply:
Ah, I see what you mean. That makes sense, will make those changes!